.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "submodules/vrs"]
 	path = submodules/vrs
 	url = https://github.com/ga4gh/vrs.git
-	branch = 2.x
+	branch = 2.0.0-ballot.2024-11
```

Comment:
I might have missed something when googling before but tags aren't supported in `.gitmodules` so I opted for the branch

Reply:
maybe we can punt this to an issue but I think commit would be appropriate (and it's effectively the same as a tag name). Otherwise there's a risk that changes to the submodule'd branch run ahead of what's expected in the superproject.